open-api/rest-catalog-open-api.yaml

```diff
@@ -173,11 +173,7 @@ paths:
         400:
           $ref: '#/components/responses/OAuthErrorResponse'
         401:
-          $ref: '#/components/responses/UnauthorizedResponse'
-        503:
```

Comment:
Shouldn't we still include 5xx responses? The oauth endpoint can also be unavailable.

Reply:
I went back and forth on that. The spec only mentions 400 and 401, and we could rely on the catch all for other response codes. That would allow us to potentially support existing OAuth servers, where the response body format won't necessarily be customizable. I wasn't sure what format we would put.